debian/hash_password.1

```diff
@@ -1,90 +1,58 @@
-.\" generated with Ronn/v0.7.3
```

Comment:
random thought: I wonder if we should generate these at build time rather than checking them in...

Reply:
That probably makes sense